packages/scandipwa/src/component/FieldSelect/FieldSelect.component.js

```diff
@@ -157,7 +157,14 @@ export class FieldSelect extends PureComponent {
                   isNotScrollable: !isScrollable
               } }
             >
-                { options.map(this.renderOption.bind(this)) }
+                <div
+                  block="FieldSelect"
+                  elem="Options-ulDiv"
```

Comment:
Please find a better name. Don't use hyphens for element names. Since hyphens are the BEM dividers in our project.